README.md

```diff
@@ -1,65 +1,87 @@
-- [x] [Guardian](https://auth0.com/docs/api/management/v2#!/Guardian/get_factors)
-- [ ] [Jobs](https://auth0.com/docs/api/management/v2#!/Jobs/get_jobs_by_id)
-- [ ] [Stats](https://auth0.com/docs/api/management/v2#!/Stats/get_active_users)
-- [x] [Tenants (Pages and Migrations)](https://auth0.com/docs/api/management/v2#!/Tenants/get_settings)
-- [ ] [Anomaly](https://auth0.com/docs/api/management/v2#!/Anomaly/get_ips_by_id)
-- [ ] [Tickets](https://auth0.com/docs/api/management/v2#!/Tickets/post_email_verification)
-- [ ] [Signing Keys](https://auth0.com/docs/api/management/v2#!/Keys/get_signing_keys)
-
-# Before you begin
-
-- This tool can be destructive to your Auth0 tenant. Please ensure you have read the documentation and tested the tool on a development tenant before using it in production.
-- Entities created using this tool may share user data with or receive user data from 3rd parties. Please see the [documentation for the individual Management endpoints](https://auth0.com/docs/api/management/v2) for details.
-
-# Documentation
-
-Please visit Auth0 Doc for this [Deploy CLI Tool](https://auth0.com/docs/deploy/deploy-cli-tool)
-
-# Integrating with popular CI/CD pipelines
-
-Please visit Auth0 Marketplace Guide for:
-
-- [Azure Pipelines](https://marketplace.auth0.com/integrations/azure-pipeline)
-- [Bitbucket Pipelines](https://marketplace.auth0.com/integrations/bitbucket-pipeline)
-- [GitLab Pipelines](https://marketplace.auth0.com/integrations/gitlab-pipeline)
-- [GitHub Actions](https://marketplace.auth0.com/integrations/github-actions)
-
-# Known issues
-
-See https://github.com/auth0/auth0-deploy-cli/issues
-
-# License
-
-MIT
+<div align="center">
+  <h1>Auth0 Deploy CLI</h1>
+</div>
+
+The Auth0 Deploy CLI is a tool that helps you manage your Auth0 tenant configuration. It integrates into your development workflows as a standalone CLI or as a node module.
+
+**Supported resource types:** actions, branding, client grants, clients (applications), connections, custom domains, email templates, emails, grants, guardian, hook secrets, hooks, log streams, migrations, organizations, pages, prompts, resource servers (APIs), roles, rules, rules configs, tenant settings, themes.
+
+## Highlights
+
+- **Multi-Environment Oriented:** Designed to help you test your applications' Auth0 integrations from feature branch all the way to production.
+- **Keyword Replacement:** Shared resource configurations across all environments with dynamic keyword replacement.
+- **Versatile:** Integrate into your CI/CD workflows either as a CLI or as a Node module.
+
+## Documentation
+
+- [Using as a CLI](#)
+- [Using as a Node Module](#)
+- [Configuring the Deploy CLI](#)
+- [Keyword Replacement](#)
+- [Incorporating Into Multi-environment Workflows](#)
+- [Excluding from Management Purview](#)
+- [Available Resource Formats](#)
+- [Terraform Provider](#)
+- [How to Contribute](#)
```

Comment:
I'd suggest adding "Issue Reporting", "What is Auth0?", and "License". See https://github.com/auth0/Auth0.swift#issue-reporting

Reply:
(to the bottom of this README). I just noticed this isn't a table of contents.